src/main/controllers/update_preference/content.ts

```diff
@@ -0,0 +1,33 @@
+import { FormContent } from '../../definitions/form';
+import { AnyRecord } from '../../definitions/util-types';
+import { isFieldFilledIn } from '../../components/form/validator';
+
+export const updatePrefFormContent: FormContent = {
+  fields: {
+    updatePreference: {
+      classes: 'govuk-radios--inline',
+      id: 'update-preference',
+      type: 'radios',
+      label: (l: AnyRecord): string => l.label,
+      values: [
+        {
+          label: (l: AnyRecord): string => l.yes,
+          name: 'radio1',
+          value: 'Yes',
```

Comment:
I think it might be better to import the Yes / No constants

Reply:
Imported the yes no constants.